frontend/src/metabase/common/components/Sidesheet/Sidesheet.tsx

```diff
@@ -0,0 +1,79 @@
+import type React from "react";
+import { t } from "ttag";
+
+import { Modal, Stack } from "metabase/ui";
+
+import Styles from "./sidesheet.module.css";
+
+type Size = "xs" | "sm" | "md" | "lg" | "xl" | "auto";
```

Comment:
Does this already exist elsewhere in the codebase?

Reply:
I don't really want to create a cross dependency between sidesheet sizes and sizes for other things. maybe we suddenly want 2xl text, but 2xl will be meaningless for sidesheets and create type errors

Reply:
I concur